### ISSUE TITLE
unexpected token error

### ISSUE THREAD

**Issue by brewster1134** [openlayers/jsdoc-plugin-typescript#13]
not sure what level of ts support this plugin has, but figured i would post my issue.

getting the error `Unexpected token, expected "]"`

my code is 
```ts
export default class Foo {
  [api: string]: Function
}
```
